README.md

```diff
@@ -1,15 +1,18 @@
 ## Ravelin Data Science Test
 
-Below are some json profiles representing fictional customers from an ecommerce company. The profiles contain information about the customer, their orders, their transactions, what payment methods they used and whether the customer is fraudulent or not. Your task is one of the following.
+Below are some json profiles representing fictional customers from an ecommerce company. The profiles contain information about the customer, their orders, their transactions, what payment methods they used and whether the customer is fraudulent or not. Your task is to:
```

Comment:
Do you think it's worth mentioning that the file is line delimited?

Reply:
I think that being able to figure out the format of whatever data is presented to you is probably something that we want to test.